src/services/stake-service.js

```diff
@@ -1,6 +1,6 @@
 import Web3 from 'web3';
-import { Token } from '../ethereum/instances/instances';
-import { hexToBytes } from '../utils/utils';
+import {Token} from '../ethereum/instances/instances';
+import {hexToBytes} from '../utils/utils';
 
 const DISPATCHER_ADDRESS = '0xAB51fBDd4Faf6c691884B3A9b475E34E2092aE81';
```

Comment:
@KPrasch what is this one?

Reply:
Yes - good catch - this is incorrect and from testnet.